pkg/cluster/k8sres.go

```diff
@@ -149,7 +149,7 @@ func fillResourceList(spec acidv1.ResourceDescription, defaults acidv1.ResourceD
 	return requests, nil
 }
 
-func generateSpiloJSONConfiguration(pg *acidv1.PostgresqlParam, patroni *acidv1.Patroni, pamRoleName string, logger *logrus.Entry) (string, error) {
+func generateSpiloJSONConfiguration(pg *acidv1.PostgresqlParam, patroni *acidv1.Patroni, pamRoleName string, logger *logrus.Entry, isNewCluster bool, initialPatroniConfig string) (string, error) {
```

Comment:
[tests](https://github.com/zalando/postgres-operator/blob/93bfed3e7562f7d6784500d7b523190b6a352e9b/pkg/cluster/k8sres_test.go#L87) also need to be updated